tutorial/markdown/mobile/sync-gateway-setup/sync-gateway-setup.md

```diff
@@ -465,7 +464,7 @@ docker-compose up -d
 
 * You can check the status of docker using either Docker Desktop or the terminal commands.  
 
-* Docker Desktop Users should see a container listing after launching the app.  The group name of the containers is the name of the directory of the code repo on your computer where you ran docker compose from. In the example it's named `android-kotlin-cbl-learning-path`.
+* Docker Desktop Users should see a container listing after launching the app.  The group name of the containers is the name of the directory of the code repo on your computer where you ran docker compose of. In the example it's named `android-kotlin-cbl-learning-path`.
```

Comment:
was the switch to 'of' intentional here? Could prob just remove the preposition.
```suggestion
* Docker Desktop Users should see a container listing after launching the app.  The group name of the containers is the name of the directory of the code repo on your computer where you ran docker compose. In the example it's named `android-kotlin-cbl-learning-path`.
```